tests/events_test.py

```diff
@@ -135,7 +135,7 @@ def test_delete_closed_events_should_delete_old_closed_event(self, tmp_path):
             events.delete_expired_events()
         assert event.id not in events.events.keys()
 
-    def test_delete_closed_events_should_not_delete_just_closed_event(self, tmp_path):
+    def test_delete_expired_events_should_not_delete_just_closed_event(self, tmp_path):
```

Comment:
"recently" or other synonym instead of "just" I think, I was confused for a second there. "newly" might work too, in contrast with the preceding test.